curriculum/challenges/english/10-coding-interview-prep/data-structures/remove-an-element-from-a-max-heap.md

```diff
@@ -27,7 +27,7 @@ The `MaxHeap` data structure should exist.
 ```js
 assert(
   (function () {
-    var test = false;
+    let test = false;
```

Comment:
I'd use const everywhere you can

Reply:
It looks like it's conditionally reassigned. TBH these tests could probably be cleaned up a bit.